docs/changelog/83839.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 83839
+summary: Deprecation info api for camel case date formats
+area: Infra/Core
+type: deprecation
+issues: []
```

Comment:
it previously had this generated too.
Not sure this was correct - hard to remove it to get CI to pass
```
deprecation:
title: Deprecation info api for camel case date formats
area: Infra/Core
details: Please describe the details of this change for the release notes. You can
use asciidoc.
impact: Please describe the impact of this change to users

```

Reply:
https://github.com/elastic/elasticsearch/pull/83839#issuecomment-1036372110
but it fails with..
```
[validate YAML][ERROR][83839.yaml][$.deprecation.area: does not have a value in the enumeration [API, Aggregation, Allocation, Authentication, CCR, Cluster, Discovery, Engine, HTTP, Highlighters, Indices, Java, License Information, Logging, Machine Learning, Mappings, Networking, Packaging, Plugins, Script Cache, Search Changes, Search, Security, Settings, Snapshot and Restore, Transform, Transport]]
[validate YAML][ERROR][83839.yaml][$.deprecation.body: is missing but it is required]
[validate YAML][ERROR][83839.yaml][$.deprecation.details: is not defined in the schema and the schema does not allow additional properties]
[validate YAML][ERROR][83839.yaml][$.deprecation.impact: is not defined in the schema and the schema does not allow additional properties]


```

Reply:
I think the details multi-line text might have issues with YML. I think you'll need ` > ` after `details:` and two space indent for each line.